src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs

```diff
@@ -26,6 +26,21 @@ public class LanguageViewLocationExpander : IViewLocationExpander
     {
         private const string ValueKey = "language";
 
+        public LanguageViewLocationExpander(
+            LanguageViewLocationExpanderOption option = LanguageViewLocationExpanderOption.Suffix)
```

Comment:
We generally avoid default-values in methods. Instead make an overload with 0-params that calls the other with the default value.